string Class
int64 id
float64 probability
int64 xmin
int64 ymin
int64 xmax
int64 ymax
int64 xc    # center x
int64 yc    # center y
int64 vx    # velocity x pix/s
int64 vy    # velocity y pix/s

